dash_prototype/main_dash.py

```diff
@@ -13,35 +13,37 @@
 app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
 
 
-import urllib.request, json
+import json
 server_url = "http://127.0.0.1:5000/"
 
+import requests
+
 def convertdate(dstring):
     return datetime.strptime(dstring, '%Y-%m-%d %H:%M:%S')
 
 def get_axis(data_url):
-    with urllib.request.urlopen(data_url) as url:
-        data = url.read().decode()
-        indexes = data.split(",")
-        if indexes and ":" in indexes[0]:
-            return list(map(convertdate, indexes))
-        else:
-            return list(map(int, indexes))
+    resp = requests.get(data_url)
+    indexes = resp.content.decode(resp.encoding).split(",")
+    if indexes and ":" in indexes[0]:
+        return list(map(convertdate, indexes))
+    else:
+        return list(map(int, indexes))
+
 
 def get_data(data_url):
-    with urllib.request.urlopen(data_url) as url:
-        data = url.read().decode()
-        return list(map(float, data.split(",")))
+    resp = requests.get(data_url)
+    data = resp.content.decode(resp.encoding)
+    return list(map(float, data.split(",")))
 
 def get_ensembles():
-    with urllib.request.urlopen(server_url+"ensembles") as url:
-        data = json.loads(url.read().decode())
-    return data["ensembles"]
+    resp = requests.get("{base}/ensembles".format(base=server_url))
```

Comment:
I think the server_url already has a ending /, but this works?

Reply:
Yes, double `//` is not an issue. However, I didn't do it on purpose, and I think it rather should be single `/`